python/ray/actor.py

```diff
@@ -319,6 +322,7 @@ def __init__(
         runtime_env,
         concurrency_groups,
         scheduling_strategy: SchedulingStrategyT,
+        enable_task_fast_fail,
```

Comment:
The naming is a bit unclear to me. Prefer something like "fail_tasks_during_restart"?